src/components/OfflineWithFeedback.js

```diff
@@ -89,6 +89,9 @@ function applyStrikeThrough(children) {
 
 const OfflineWithFeedback = (props) => {
     const hasErrors = !_.isEmpty(props.errors);
+    // Some errors might not have error messages when we want to apply opacity without showing any errors.
```

Comment:
```suggestion

    // Some errors have a null message. This is used to apply opacity only and to avoid showing redundant messages.
```
maybe this is better?